### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by justineechen** [OpenLiberty/draft-guide-istio#23]
- [x] Record the time it takes to go through the entire guide
approx ~28minutes
- [x] Test each step in the guide is clear to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- `http://<ip-address>:<port>/hello` gave me "This site can't be reached <ip-address> took too long to respond", I made sure that `kubectl get deployments` were available before trying to navigate to the url. 

- [x] Resolve anything usability issues (confusing or hard to follow)

- It might be nice to reiterate the port number for the user when they need to navigate to `http://<ip-address:<port>/hello` 

- [x] Achieve basic understanding/working knowledge of the topic in the guide

**Comment by proubatsis** [comment]
@justineechen Some questions about the feedback:

> http://:/hello

Why did you do this?

> after running kubectl delete -f install/kubernetess/helm/istio/templates/crds.yaml at the last section

The instruction before this stated `Navigate to the directory where you extracted Istio and delete the Istio resources from the cluster.`, did you perform this step?

**Comment by justineechen** [comment]
@proubatsis 
sorry, the URL wasn't backtracked it's `http://<ip-address>:<port>/hello` 

And no that is my mistake, I misread that line so the error shouldn't have shown.

**Comment by proubatsis** [comment]
PR Merged
